### DISCUSSION TITLE
Tillage type

### DISCUSSION THREAD

**Discussion by daSilva5** [APSIMInitiative/ApsimX#9304] [Q&A]
Hello APSIM community,
Is there a reason NextGen does not have tillage options (e.g. rip)?
Also how does tillage interact with simulation outcomes, such as yield?

Cheers

**Comment by hol353** [comment]
APSIM only needs to know the fraction of surface residue to incorporate into the soil and to what depth of soil does the incorporation happen. It doesn't have tillage implements. To simulate a rip you would need to configure this yourself in 'tillage on a fixed date':

![image](https://github.com/user-attachments/assets/0cabb6f6-4f55-432b-9e58-90e2358d7e26)

In simple terms, incorporating residues into the soil speeds up their breakdown, releasing nutrients into the soil and making them available to the plant to uptake sooner. This might affect yield of the subsequent crop. Alternatively, leaving residues on the soil surface reduces runoff, allowing more water to infiltrate the soil. This might also affect yield of the subsequent crop.

**Reply by daSilva5** [reply]
Thank you for your answer. 
How should I specify a tillage type? In APSIM classic there are options for it. When you select tillage on a fixed date, you can select tillage types from a field.
Also, I would like to know what APSIM does when there is a sequence of tillage operations. Is the effect additive, multiplicative, or APSIM just accounts for the last one in the list? 


Kind regards,
Isis

**Comment by hol353** [comment]
You cannot add a tillage type. You add one or more 'Tillage on a fixed date' models and configure it (enter a fraction of residue to incorporate and a depth of incorporation). You can also do this via an operations model.

If you specify multiple tillages on a day, then APSIM will iterate through each one and apply them, one at a time.

**Reply by daSilva5** [reply]
Thank you!
